src/Microsoft.Health.SqlServer/ManagedIdentityAccessTokenHandler.cs

```diff
@@ -1,29 +1,24 @@
-﻿// -------------------------------------------------------------------------------------------------
+// -------------------------------------------------------------------------------------------------
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
 using System.Threading;
 using System.Threading.Tasks;
-using EnsureThat;
-using Microsoft.Azure.Services.AppAuthentication;
```

Comment:
Can you remove the associated NuGet package too?

Reply:
I still don't see a change to remove it from both the `csproj` file and the `Directory.packages.props`. Presumably this is `Microsoft.Azure.Services.AppAuthentication`?